config/environments/production.rb

```diff
@@ -83,7 +87,7 @@
 
   # Use a different logger for distributed setups.
   # require 'syslog/logger'
-  config.logger = GELF::Logger.new("graylog", 12219, "WAN", { :facility => "Main App Server" })
+  config.logger = GELF::Logger.new("graylog", 12_219, "WAN", facility: "Main App Server")
```

Comment:
die Port Nummer so anzugeben funktioniert vermutlich, aber ob es wirklich notwendig ist, weiß ich nicht

Reply:
ach krass...durchs linting. mal sehen obs klappt.
aber zeigt uns auch, dass wir in den linter configs sowas ausstellen müssten!